webviz_subsurface/plugins/_well_cross_section_fmu.py

```diff
@@ -118,25 +135,26 @@ def __init__(
         self.zonemin = zonemin
         self.segyfiles = [] if segyfiles is None else [str(segy) for segy in segyfiles]
         self.zonelog = zonelog
-
+        self.plotly_theme = app.webviz_settings["theme"].plotly_theme
         self.colors = (
             colors
             if colors
-            else [
-                "#1f77b4",  # muted blue
-                "#ff7f0e",  # safety orange
-                "#2ca02c",  # cooked asparagus green
-                "#d62728",  # brick red
-                "#9467bd",  # muted purple
-                "#8c564b",  # chestnut brown
-                "#e377c2",  # raspberry yogurt pink
-                "#7f7f7f",  # middle gray
-                "#bcbd22",  # curry yellow-green
-                "#17becf",  # blue-teal
-            ]
+            else self.plotly_theme["layout"].get(
+                "colorway",
+                [
+                    "#1f77b4",  # muted blue
+                    "#ff7f0e",  # safety orange
+                    "#2ca02c",  # cooked asparagus green
+                    "#d62728",  # brick red
+                    "#9467bd",  # muted purple
+                    "#8c564b",  # chestnut brown
+                    "#e377c2",  # raspberry yogurt pink
+                    "#7f7f7f",  # middle gray
+                    "#bcbd22",  # curry yellow-green
+                    "#17becf",  # blue-teal
+                ],
+            )
         )
-        self.plotly_theme = app.webviz_settings["theme"].plotly_theme
```

Comment:
This change of behavior for colors should probably be in a separate PR, just something I realized when writing the docs. Anyways: seems like there is a bug that overwrites the input colors with theme colors if theme colors exist. Should rather use theme colors only if input colors are None.  Will revert this and write a separate issue before a merge.

Reply:
Feels like I changed the style a bit while working through this. Especially when it comes to the "Data input" section. Not entirely sure what the best format is. 

Also a lot of notes, `code styled words`, code blocks and hyperlinks make it look a bit messy (especially since they are all colored). Not sure if clean, or "clear" (with e.g. `code styled words`) are the right way to go. Many colors and fonts make it a bit tiring to read it in my opinion.

Reply:
This is moved to #382